src/test/scala/com/fulcrumgenomics/vcf/filtration/ReadEndSomaticVariantFilterTest.scala

```diff
@@ -42,15 +43,143 @@ class EndRepairArtifactLikelihoodFilterTest extends UnitSpec {
     Genotype(as, "s1", as.toIndexedSeq, phased=false)
   }
 
-  "appliesTo" should "return false for any event that is not a SNP" in {
-    val filter  = new EndRepairArtifactLikelihoodFilter()
+  "isPointMismatch" should "return true for any SNV" in {
+    val bases = Seq("A", "C", "G", "T")
+    for (ref <- bases; alt <- bases; if ref != alt) yield isPointMismatch(singleGenotype(ref, alt)) shouldBe true
+  }
+
+  it should "return false for any event that is not a SNP" in {
+    ReadEndSomaticVariantFilter.isPointMismatch(singleGenotype("A",  "AT")) shouldBe false // Insertion
+    ReadEndSomaticVariantFilter.isPointMismatch(singleGenotype("TA", "A"))  shouldBe false // Deletion
+    ReadEndSomaticVariantFilter.isPointMismatch(singleGenotype("AT", "GC")) shouldBe false // MNP
+  }
```

Comment:
I think you need a more comprehensive test set here, and to use something other than the existing `singleGenotype` method to generate them.  As noted above the method isn't inspecting the genotype, but `singleGenotype` always sets the variant alleles and the genotype's alleles to the same values, so the tests didn't catch that.  
Please add some tests with > 1 ALT allele and a variety of het and hom genotypes.

Reply:
resolved